Packs/FWToCDLMonitoring/Scripts/FWToCDLMonitoring/FWToCDLMonitoring.py

```diff
@@ -0,0 +1,109 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+import datetime as dt
+
+# Cortex Data Lake FW log monitoring monitoring
+# PRE_REQUISITE: Enable CDL integration and/or Panorama integration (if the automated creation of a FW list is expected)
+
+
+CORTEX_BRAND = "Cortex Data Lake"
+NO_LOG_ANSWER = "### Logs traffic table\n**No entries.**\n"
+# Looking for the last 12 hours of logs
+LOG_MONITORING_PERIOD = 12
+
+# Gather existing instances
+allInstances = demisto.getModules()
+
+# Look for active Cortex Data Lake instance
+cortex_found = 0
+for instance in allInstances:
+    if allInstances[instance]['brand'] == CORTEX_BRAND and allInstances[instance]['state'] == 'active':
+        cortex_found = 1
```

Comment:
To save time you can break after you find the instance.

```suggestion
        cortex_found = 1
        break
```